locale/zh/messages.json

```diff
@@ -738,7 +740,7 @@
   "Scheduled": "已计划",
   "Search": "搜索",
   "Search Results": "搜索结果",
-  "Search/bind": "",
+  "Search bind": "搜索 绑定",
```

Comment:
do these relate to this PR? 🤔

Reply:
Like I said, no, but we lost this translation somewhere, so I added it again.

Reply:
ah, missed that in the PRs description. If thats the case, please make them a separate commit. 🙇